examples/meta/src/evaluation/cross_validation_timeseries_split.sg

```diff
@@ -6,16 +6,16 @@ RegressionLabels labels(f_labels)
 
 #![set parameters]
 int num_subsets = 5
-int min_subset_size = 4
+int future_steps = 4
 TimeSeriesSplitting splitting(labels, num_subsets)
-splitting.set_min_subset_size(min_subset_size)
+splitting.set_min_future_steps(future_steps)
 #![set parameters]
 
 #![build subsets]
 splitting.build_subsets()
 #![build subsets]
 
-#![generate subsets and inverse (aka test labels and train labels)]
-IntVector test_labels_indices = splitting.generate_subset_indices(1)
+#![generate subsets and inverse (aka vaildation set and train set)]
```

Comment:
This is a tag for the cookbooks, not free form comment. So pls make it super short (see other examples)